sdk/iot_hub/src/service/requests/get_twin.rs

```diff
@@ -27,7 +26,9 @@ where
     };
 
     let request = service_client.prepare_request(&uri, Method::GET);
-    let request = request.body(azure_core::EMPTY_BODY)?;
+    let request = request
+        .body(azure_core::EMPTY_BODY)
+        .map_kind(ErrorKind::Io)?;
```

Comment:
Is this `map_kind(...)` required?  Other similar functions seem to build with `.body(azure_core::EMPTY_BODY)?`

Reply:
Good catch. Removed.